### ISSUE TITLE
Small errors in docs

### ISSUE THREAD

**Issue by laradevitt** [horacioh/gatsby-theme-instagram#6]
Hi, @horacioh!

Is there a way to submit a PR to a project forked from another? This is a new situation for me. I was hoping to submit a PR to fix a couple of errors in your [theme docs](https://github.com/horacioh/gatsby-theme-instagram/tree/master/theme) but was unable to do so. 

There are just a couple of issues that I ran into:

1. In the *Public scraping for posts* example, the line with `type` property should end with a comma.

2. In your *Hashtag scraping* example, I believe the plugin should be `@horacioh/gatsby-theme-instagram`, not `gatsby-source-instagram`. At least that is what worked for me.

Great theme! 

Cheers -

**Comment by horacioh** [comment]
hey @laradevitt!
first of all, THANK YOU for filing this issue 🙏 

and yes, you are absolutely right on both issues!, so let me try to help you to do this contribution.

- I think you can fork my repo right now. are you able to see a `fork` button on the top right of my repo? if you do, you can press that and it sould let you fork my repo :)
- if that does not work, you can go to the [`README.md`](https://github.com/horacioh/gatsby-theme-instagram/blob/master/theme/README.md) file, and press on the pencil icon on the top right corner of the file container
<img width="1154" alt="Screen Shot 2019-08-12 at 4 06 24 AM" src="https://user-images.githubusercontent.com/725120/62843013-e4c9a200-bcb6-11e9-99b8-070f24aa6412.png">

after you edit the file, scroll all the way down and select the second radio button and press the `Propose file changes` button:
<img width="1181" alt="Screen Shot 2019-08-12 at 4 09 41 AM" src="https://user-images.githubusercontent.com/725120/62843041-2c502e00-bcb7-11e9-8f0d-f40b3a9f4c32.png">

please let me know if this helps, and thanks again to point it out this fixes!

**Comment by laradevitt** [comment]
Hi, @horacioh. Thanks for this, but what I meant was, I am unable to fork in the usual way (as you describe) because your repo is a fork of *jlengstorf/gatsby-theme-jam-example* and I already have my own fork of that project (see screenshot). 

Sorry if I wasn't clear! 

![github-fork](https://user-images.githubusercontent.com/13952507/62858932-ff047f80-bcb8-11e9-948e-c4729e288073.jpg)

Even though it says "You've already forked gatsby-theme-instagram", mine is actually another fork of *jlengstorf/gatsby-theme-jam-example*. In order to submit a PR to your project I would have to delete my fork.

I found confirmation here: https://stackoverflow.com/a/6676221/4483099

Cheers -

**Comment by horacioh** [comment]
haha this is clearly a corner case on guthub!!

maybe you can try de second option? (edit in github)

if that does not work, i can give you access as a maintainer

**Comment by horacioh** [comment]
look at the recommendation on this thread: https://twitter.com/hhg2288/status/1160879177513484288?s=20

you can:
- add my repo as another remote
- create a new branch on your local repo from my fork
- update the code
- submit a PR from the new branch to my repo

this should work like a charm!!

let me know if that works!

**Comment by laradevitt** [comment]
Hey, @horacioh - I think those recommendations are for you. :wink: They explain how to "unfork" your repo. It seems to me like the way to go if you don't want others to have the same issue. 

I noticed that quite a few Theme Jam projects are still forked from the example theme, but many are not.

**Comment by horacioh** [comment]
the recommendation from Alberto is for you too!

this are the steps:
1. Add my repo as a remote to your fork
```
cd go-to-gatsby-theme-molly
```

2. add my repo as a remote to your repo:
```
git remote add horacio git@github.com:horacioh/gatsby-theme-instagram.git
```

3. create a new branch in your repo, from my repo
```
git checkout -b gatsby-theme-instagram horacio/master
```

doing this, will pull my repo into your repository, you can make changes, push this new branch to your repo and then you can do a PR :)

if this is too much I understand!, I can make myself the changes, but I wanted you to have the contribution since you point it out the errors.

let me know if this helps!

**Comment by laradevitt** [comment]
Sure, I'll give it a try just for fun. :smiley: It might be tomorrow, though.

**Comment by laradevitt** [comment]
I thought it worked, but there's something going on. "All checks failed". 😬 

In the set-up, a couple more steps were required after adding the remote.

After

```cli
git remote add horacio git@github.com:horacioh/gatsby-theme-instagram.git
```

then do 

```cli
git remote update
```

and

```cli
git fetch
```

to update all tracked branches. 

Source: [Git: solving error “Cannot update paths and switch to branch”](https://medium.com/@dianaarnos/git-solving-error-cannot-update-paths-and-switch-to-branch-9f751d6493a3)

Then I was able to run

```cli
git checkout -b gatsby-theme-instagram horacio/master
```

**Comment by horacioh** [comment]
solved with #7 👍 
thanks @laradevitt !!
